src/v2/view-builder/views/IdentifierView.js

```diff
@@ -113,15 +113,23 @@ const Body = BaseForm.extend({
     const passwordExplainLabeli18nKey = 'primaryauth.password.tooltip';
 
     const newSchemas = schemas.map(schema => {
-      const newSchema = { ...schema };
+      let newSchema = { ...schema };
 
       if (schema.name === 'identifier' && isCustomizedI18nKey(identifierExplainLabeli18nKey, settings)) {
-        newSchema.explain = loc(identifierExplainLabeli18nKey, 'login');
-        newSchema['explain-top'] = true;
+        newSchema = {
+          ...newSchema,
+          explain: loc(identifierExplainLabeli18nKey, 'login'),
+          'explain-top': true,
+          'data-se': 'username-form-container',
```

Comment:
- `o-form-fieldset-${newSchema.name}`?
- turns out this will override existing `data-se=o-form-fieldset`.  seems not really ideal.